src/constants/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * MDM centralised constants, referred as `@ukef/constants`.
```

Comment:
I think this should say `TFS centralised constants` instead

Reply:
Good spot, have renamed.